docker/build.sh

```diff
@@ -119,18 +132,32 @@ build_images() {
         --set "*.args.BASE_IMAGE=$base_image" \
         --set "*.args.SETUP_ARGS=$setup_args" \
         --set "*.args.LIB_DIR=$lib_dir" \
+        --set "base.tags=ghcr.io/autowarefoundation/autoware:latest-base" \
         --set "devel.tags=ghcr.io/autowarefoundation/autoware:latest-devel$image_name_suffix" \
         --set "prebuilt.tags=ghcr.io/autowarefoundation/autoware:latest-prebuilt$image_name_suffix" \
         --set "runtime.tags=ghcr.io/autowarefoundation/autoware:latest-runtime$image_name_suffix" \
         "${targets[@]}"
     set +x
 }
 
+# Remove temporary source directory
+remove_tmp_src() {
+    rm -rf "$WORKSPACE_ROOT/docker/src"
+}
+
+# Remove dangling images
+remove_dangling_images() {
+    docker image prune -f
+}
+
 # Main script execution
 parse_arguments "$@"
 set_cuda_options
 set_build_options
 set_platform
 set_arch_lib_dir
 load_env
+clone_repositories
```

Comment:
@oguzkaganozt may I please check with you what is the purpose of this step? 
In the [Dockerfile](https://github.com/autowarefoundation/autoware/blob/main/docker/Dockerfile#L46) we are still copying the `src` that is under `autoware` not the `src` that is under `autoware/docker`.

Reply:
Thank you for your review. I think it is a mistake and I fix it by https://github.com/autowarefoundation/autoware/issues/4894.